app/config_objects/per_district.rb

```diff
@@ -117,15 +129,15 @@ def enable_counselor_based_feed?
   # and our communication with district authentication systems
   # to always be in terms of full email addresses with domain
   # names.
-  def from_import_login_name_to_email(login_name)
+  def from_educator_row_to_email(row)
     if @district_key == SOMERVILLE
-      login_name + '@k12.somerville.ma.us'
+      row[:login_name] + '@k12.somerville.ma.us'
     elsif @district_key == NEW_BEDFORD
-      login_name + '@newbedfordschools.org'
+      row[:login_name] + '@newbedfordschools.org'
     elsif @district_key == BEDFORD
-      login_name + '@bedfordps.org'
+      row[:email]
```

Comment:
This assumes that Bedford is sending us an educators file with an "email" column to their SFTP box.

Will was able to create this file in a one-time export, but the header was named "Email" instead of "email", and he hasn't sent it to the SFTP site through the cron job yet. 

These are small details but they all need to be in place before the Bedford educator import can run successful.

Reply:
Update: Will confirmed that the file will be updated in SFTP tomorrow, and that the header will be lowercased.

Reply:
Another update: I realized this is a blocker for importing educator records, but not a blocker for creating a developer user to test with.